packages/@snowpack/plugin-babel/plugin.js

```diff
@@ -1,11 +1,27 @@
-const babel = require("@babel/core");
+const fs = require('fs');
+const babel = require('@babel/core');
+
+module.exports = function plugin(_, options) {
+  // options validation
+  if (options) {
+    if (typeof options !== 'object') throw new Error(`options isn’t an object. Please see README.`);
+    if (options.input && !Array.isArray(options.input))
+      throw new Error(
+        `options.input must be an array (e.g. ['.js', '.mjs', '.jsx', '.ts', '.tsx'])`,
+      );
+    if (options.input && !options.input.length)
+      throw new Error(`options.input must specify at least one filetype`);
+  }
```

Comment:
`@snowpack/plugin-babel` now accepts `input: []` as a configurable option, in case users want to run more things through.

Reply:
It also validates its options, and throws a helpful error message to the user.

Reply:
(for later) I'd love to look into supporting `input` globally/automatically for build plugins, so that every plugin author doesn't need to add this support / its consistent.